easybuild/easyconfigs/j/Java/Java-15.0.2.eb

```
@@ -0,0 +1,19 @@
+#
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+#
+name = 'Java'
+version = '15.0.2'
+
+homepage = 'https://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://download.java.net/java/GA/jdk15/779bf45e88a44cbd9ea6621d33e33db1/36/GPL/']
+sources = ['openjdk-%(version)s_linux-x64_bin.tar.gz']
+checksums = ['91ac6fc353b6bf39d995572b700e37a20e119a87034eeb939a6f24356fbcd207']
+
```

```suggestion
sanity_check_commands = ["java -version"]

```

This'll help me when I accidentally install it on our IBM POWER9 - as the sanity check fails.